webapp/channels/src/components/admin_console/session_length_settings.tsx

```diff
@@ -7,33 +7,51 @@ import {FormattedMessage} from 'react-intl';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message';
 import * as Utils from 'utils/utils';
 
-import AdminSettings from './admin_settings';
+import AdminSettings, {BaseState, BaseProps} from './admin_settings';
 import BooleanSetting from './boolean_setting';
-import SettingsGroup from './settings_group.jsx';
+import SettingsGroup from './settings_group.js';
 import TextSetting from './text_setting';
+import { DeepPartial } from 'redux';
```

Comment:
Can you run `make check-style`? because order of the import and spaces between curly braces are not accepted by Mattermost type checker.